README.md

```diff
@@ -19,3 +19,43 @@ MiSeq_runs_basespace.sh
 ## daily_SARS-CoV-2_metadata.sh
 
 daily_SARS-CoV-2_metadata.sh checks the date every 4 hours. If it is a new day, this script runs the Dripping Rock nextflow workflow to get the daily SARS-CoV-2 metadata file. On Sundays, it also takes the latest 20 runs and 10 random runs and creates a phylogenetic tree. This is a script that is run in a screen on the Production account.
+
+## ICA Automation Scripts
+These scripts are intended to make the running of analyses on ICA for production purposes automated as well as downloading the most common files that need to be reviewed by human eyes. Each is written in python and uses the icav2 command line tool provided by Illumina. These are intended to run on screens on the linux work station but wherever they are ran icav2 must already be installed and configured on that system.
+
+### Usage
+These scripts run on infinite loops that should be ran on a screen in the background of the linux workstation.
+```
+python auto_Granduer_ICA.py
+```
+```
+python auto_mycosnp_ICA.py
+```
+
+### mycosnp
+Currently this pipeline is only ran on C. auris samples. These samples are usually uploaded directly into ICA by the Illumia instrument after BSSH creates FASTQ files.
+The automation script runs each time these files make their way to ICA. It produces the samplesheet, including control samples, needed for the analysis and uploads it to ICA. It then starts the analysis.
+
+### Grandeur
+We are currently uploading sequencing runs that use granduer using the General_LW_scripts/MiSeq_runs_basespace.sh. This script looks for those uploads then sees if they have already had analysis preformed on ICA. If not they start that analysis.
+
+### Alternatives if Scripts Fail
+
+#### Manually Starting Analysis Using icav2 CLI
+This is the basic command for starting an analysis with icav2: ```icav2 projectpipelines start nextflow [pipeline id] or [code] [flags] ```. The pipeline is the name of the pipeline, i.e. UPHL-BioNGS_Granduer_V2. Then you must provide the inputs with this syntax ```--input parametercode:dataId,dataId,dataId,... ``` Finally you must provide a name ```--user reference $SEQUENCING_RUN_PIPELINE_Analysis```
+To find data Ids use ```icav2 projectdata list ```.
+##### mycosnp
+Too hard without scripting. The automation script creates an icav2 command; using this command to troubleshoot is probably the best solution. 
```

Comment:
This is a problem. We don't want to be held hostage by script that only runs in the background.